supplementary_style_guide/style_guidelines/grammar.adoc

```diff
@@ -128,4 +128,30 @@ Avoid long introductions and unnecessary context. Shorten unnecessarily long sen
 === Principle 5: Error recovery, verification, and troubleshooting
 Recognize that people make mistakes and need to verify that they have completed a task. Be sure to include troubleshooting, error recovery, and verification steps.
 
+[[users]]
+== Users
+In most cases, the word "user" refers to a person or a person's user account, and therefore would be considered animate. In these cases, use animate personal pronouns such as "who".
+
+However, in certain technical cases, these users are not persons but instead user accounts or more abstract concepts. For example, Linux users such as root and guest do not relate to any person. Applications and services might run as specific Linux users with no person controlling them. SELinux users such as `user_u` or `sysadm_u` are identifiers of one or multiple Linux users for access control purposes.
+
+In these specific cases, and only if you cannot write around it, you can refer to these inanimate users with inanimate personal pronouns such as "that".
+
+.Examples
+* Animate user
++
+image:images/no.png[no] Experienced _users that_ can configure their own systems...
```

Comment:
```suggestion
image:images/no.png[no] Experienced _users that_ can configure their own systems....
```

Close ellipses trailing sentence with a period.

Same suggestion for the next example.